src/components/weather/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from "react";
+import Loader from "react-loader-spinner";
+import { ToastContainer, toast } from "react-toastify";
+import "./styles.css";
+import "react-loader-spinner/dist/loader/css/react-spinner-loader.css";
+import "react-toastify/dist/ReactToastify.css";
+
+import CurrentWeather from "./currenetWeather.jsx";
+import Forecast from "./forecast.jsx";
+
+import { fetchWeather } from "api/weather";
+import { isEmpty } from "utils/obj";
+
+function App() {
+  const [current, setCurrent] = useState({}),
+    [daily, setDaily] = useState([]),
+    [lat, setLat] = useState(null),
+    [lon, setLon] = useState(null),
```

Comment:
What are the benefits of segmenting out lat and lon into separate ```useState``` functions? Curious if you tried making this one ```useState``` with an object like: ```{ lat: null, lon: null }```.